internal/pkg/plugin/gitlabci/java/options.go

```diff
@@ -0,0 +1,124 @@
+package java
+
+import (
+	"fmt"
+	"html/template"
+
+	"github.com/spf13/viper"
+)
+
+// Options is the struct for configurations of the gitlabci-java plugin.
+type Options struct {
+	PathWithNamespace string   `validate:"required"`
+	Branch            string   `validate:"required"`
+	BaseURL           string   `validate:"omitempty,url"`
+	Package           *Package `validate:"required"`
+	Build             *Build   `validate:"required"`
+	Deploy            *Deploy  `validate:"required"`
+}
+
+type BaseOption struct {
+	Enable bool `validate:"required"`
```

Comment:
If set to `required`, then `Enable` must be `true`.

`required` requires a non-zero value, while `false` is the zero value of `bool`.